Kalk88/GrowBud
# feat: 🎸 Update schedules

## **Description**
When a schedules next time to water have passed it should be updated
with the interval associated with that schedule. This review creates a
funtion that when called finds all schedules with a time to water in the
past and updates them.

## **How to test?**
This is running on the default environment, create a schedule, call the function and note the new date for the schedule. Call it again and nothing should be updated.
